porcupine/plugins/comment_selected_lines.py

```diff
@@ -34,12 +52,10 @@ def comment_or_uncomment(tab: tabs.FileTab, pressed_key: str | None = None) -> s
     all_linenos = set(range(start, end))
     commented = {
         lineno
-        for lineno, line in enumerate(
+        for lineno, line_text in enumerate(
             tab.textwidget.get(f"{start}.0", f"{end}.0").splitlines(), start
         )
-        # Ignore '# blah' comments because they are likely written by hand
-        # But don't ignore indented '#    blah', that is most likely by this plugin
-        if line.startswith(comment_prefix) and not re.match(r" [^ ]", line[len(comment_prefix) :])
+        if already_commented(line_text, comment_prefix)
```

Comment:
Why are comments getting ignored??

Reply:
Akuli wanted manually added comments to be ignored when the keybinding is used.

Reply:
Ummmm ok. He does what he does ig.

Reply:
To be fair, he is also the project owner, so :)